sources/packages/api/src/route-controllers/confirmation-of-enrollment/confirmation-of-enrollment.controller.ts

```diff
@@ -98,6 +114,13 @@ export class ConfirmationOfEnrollmentController {
     @Param("applicationId") applicationId: number,
   ): Promise<number> {
     try {
+      const approvedOrDeniedCOE =
+        await this.disbursementScheduleService.getModifiedCOE(applicationId);
```

Comment:
Instead of getModifiedCOE can we call the getFirstCOEOfApplication here and do the validation?

Reply:
Modified to use getfirstCOE as discussed. The getModifiedCOE has been removed in service.